bin/cozy-light

```diff
@@ -59,7 +69,7 @@ program
   .action(actions.enable);
 
 program
-  .command('install-distro [distro]')
+  .command('install-distro <distro>')
```

Comment:
Distro is optional. I'm not sure we should write the argument that way. When no distro is given, it displays the list of available distros.